lib/Buzz/Browser.php

```diff
@@ -40,43 +29,43 @@ class Browser
     /** @var RequestInterface */
     private $lastRequest;
 
-    /** @var MessageInterface */
+    /** @var ResponseInterface */
     private $lastResponse;
 
-    public function __construct(ClientInterface $client = null, FactoryInterface $factory = null)
+    public function __construct($client = null)
```

Comment:
accepting anything ? really

Reply:
Yeah.. Awful. We should do the new BuzzClientInterface introduced here: https://github.com/kriswallsmith/Buzz/pull/270